providers/ms365/resources/ms365.lr

```diff
@@ -1100,6 +1100,8 @@ ms365.sharepointonline {
   spoTenantSyncClientRestriction() dict
   // SharePoint Online tenant sites
   spoSites() []ms365.sharepointonline.site
+  // The default link permission for the tenant
+  defaultLinkPermission() []string
```

Comment:
As far as I can see the specification this can only return one value, so this should not be a string slice